src/algebra/direct_sum/basic.lean

```diff
@@ -76,6 +76,11 @@ def mk (s : finset ι) : (Π i : (↑s : set ι), β i.1) →+ ⨁ i, β i :=
 def of (i : ι) : β i →+ ⨁ i, β i :=
 dfinsupp.single_add_hom β i
 
+lemma of_congr {i i' : ι} (h1 : i = i') {x : β i} {y : β i'}
```

Comment:
Elsewhere I've been using [`direct_sum.of_eq_of_graded_monoid_eq`](https://leanprover-community.github.io/mathlib_docs/algebra/direct_sum/ring.html#direct_sum.of_eq_of_graded_monoid_eq) instead of needing this lemma.

Reply:
done